### PR TITLE
chore: update read me and corrects release version

### PR DESCRIPTION
**Pull request by Rj-Rahul-96**

### CODE REVIEW

File: .nuget/SourceGrid.nuspec [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
   <metadata>
-    <id>SourceGrid</id>
-    <version>5.0.0</version>
+    <id>Siemens.SourceGrid</id>
```
Comment (RenishPalliyath) [inline .nuget/SourceGrid.nuspec]: make it all small letter

**Review by RenishPalliyath — COMMENTED** [review]
modify the changelog.md files too

**Comment by Rj-Rahul-96** [comment]
> modify the changelog.md files too

updated

File: .nuget/SourceGrid.nuspec (hunk 1 above)
Comment (Rj-Rahul-96) [reply .nuget/SourceGrid.nuspec]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,12 +2,6 @@
...
-- Enhancement: Smooth horizontal and vertical scrolling
-- Enhanced Freeze panes(FixedRow and FixedColumn) and made it independent of Header row\column count
-- Introduced a boundary(defined by user) to stop auto scrolling
-- Filter row support in DataGrid
-- Support for Drag and drop of cells
-- Performance improvement while loading grid- CreateControl
-- Selectable readonly cells
-- Introduced a disabled cell mode
-- Fixed bugs in clipboard, spanning etc.
+- Migrate to .net framework 4.8
```
Comment (RenishPalliyath) [inline CHANGELOG.md]: please add the issue number here like you did for 3rd item

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # SourceGrid
 
-## Major Changes
+## [1.0.0] - 2019-04-30
```
Comment (RenishPalliyath) [inline CHANGELOG.md:15]: one last change..add the new changes on top.. else we have to scroll down to read :)

File: CHANGELOG.md (hunk 2 above)
Comment (Rj-Rahul-96) [reply CHANGELOG.md]: Added

File: CHANGELOG.md (hunk 3 above)
Comment (Rj-Rahul-96) [reply CHANGELOG.md:15]: Done

**Review by RenishPalliyath — APPROVED** [review]

**Pull request merged**